code/index.html

```diff
@@ -2,21 +2,129 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Project 2: The Review</title>
     <link rel="stylesheet" href="./style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" type="image/png" href="images/Logga_favicon.png"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Knewave&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;600&family=Source+Serif+Pro:wght@300;600&display=swap" rel="stylesheet">
+    
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
-    </header>
+      <header class="header">
+        
+        <nav>
+            <a href="#">Latest</a>
+            <a href="#">Art</a>
+            <img class="logo" src="./Logga_R.jpg" alt="logotype" />
+            <h1>The Review </h1>
+            <a href="#">Pets</a>
+            <a href="#">World</a>
+        </nav>
+      </header>
 
-    <section class="big-news">
+
+    <Section class="big-news">
       <!-- The big news content -->
```

Comment:
Maybe have the comment before the section starts, so it's easier to see where it begins.